matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/content/UploadContentWorker.kt

```diff
@@ -288,7 +293,7 @@ internal class UploadContentWorker(val context: Context, params: WorkerParameter
                                       newImageAttributes: NewImageAttributes?): Result {
         notifyTracker(params) { contentUploadStateTracker.setSuccess(it) }
         params.localEchoIds.forEach {
-            updateEvent(it.eventId, attachmentUrl, encryptedFileInfo, thumbnailUrl, thumbnailEncryptedFileInfo, newImageAttributes)
+            updateEvent(it.eventId, attachmentUrl, encryptedFileInfo, thumbnailUrl, thumbnailEncryptedFileInfo, newImageAttributes, params.attachment.size)
```

Comment:
I do not understand why you are adding a param here, can't we use `NewImageAttributes.newFileSize`

Maybe rename `NewImageAttributes` with `NewAttachmentAttributes` by the same occasion.

Reply:
Done.